### DISCUSSION TITLE
Disabling Alt [hjkl]  and remapping entering search mode not working

### DISCUSSION THREAD

**Discussion by mbaugustyn** [zellij-org/zellij#3240] [Q&A]
Hi,

1. I tried to disable shortcuts Alt hjkl (they conflict with my helix keybindings).
The problem is disabling is not working - you can even see them listing in the help section on the bottom.
When I remap them to something else - it is succesfull and they dissapear from the help section.
2. I tried to remap Cntr s (enter search mode) to Cntr x. But it is not working as suspected. When i enter Ctrl x, i enter search mode but it is not working properly, and when i enter "Cntr s" I go into a weird mode in which nothing is working and I have to enter "Cntr C" to exit from it.
My config:
![image](https://github.com/zellij-org/zellij/assets/32998370/5b9e551a-99f4-4843-9f61-223ed89a36ec)

**Comment by cristiand391** [comment]
1. this config should disable `Alt + hjkl` in **all** modes
```
keybinds {
    unbind "Alt h"
    unbind "Alt j"
    unbind "Alt k"
    unbind "Alt l"
}
```
docs: https://zellij.dev/documentation/keybindings-binding#overriding-keys

are you restarting zellij after the config changes? live config reload isn't supported yet.

2. are you sure it's the new keybindings aren't conflicting with other defaults?

**Reply by mbaugustyn** [reply]
1. I am restarting after each config change. I already tried your suggestion and it is not working  for me (I do not know why - maybe someone can explain) - is it working for you? What worked for me was unbinding Alt + hjkl in normal mode specifically.
2. You might be right - I did not think about that. They may be conflicting with default bash config.
